cdap-gateway/src/main/java/co/cask/cdap/gateway/router/handlers/SecurityAuthenticationHttpHandler.java

```diff
@@ -80,6 +83,23 @@ public SecurityAuthenticationHttpHandler(String realm, TokenValidator tokenValid
     this.discoveryServiceClient = discoveryServiceClient;
     this.discoverables = discoveryServiceClient.discover(Constants.Service.EXTERNAL_AUTHENTICATION);
     this.configuration = configuration;
+    this.bypassPattern = createMatcher(configuration.get(Constants.Security.Router.BYPASS_AUTHENTICATION_REGEX));
```

Comment:
This is a configuration for the router. So I believe it belongs in CConf. But it's a good question: do we have a guideline as to what belongs in what config? The security.enabled, for example, is in the cConf,  not the sConf.